### ISSUE TITLE
E9 Literal definition of 'gcis:Table'

### ISSUE THREAD

**Issue by xgmachina** [USGCRP/gcis-ontology#82]
http://data.globalchange.gov/gcis.owl#d4e2126 (or http://data.globalchange.gov/gcis.owl#Table) -- Revise the definition to exclude use of the term “list” (too-CS specific).

**Comment by zednis** [comment]
Current class definition:

```
gcis:Table a owl:Class ;
    rdfs:label "Table" ;
    rdfs:comment "A list of facts or numbers arranged in a special order, usually in rows and columns." ;
    rdfs:subClassOf prov:Entity .
```

The second definition of 'table' from the New Oxford American Dictionary aligns pretty closely with the definition we have been using.  I am fine with keeping what we have.

table |ˈtābəl|
noun
1 a piece of furniture ...
2 a set of facts or figures systematically displayed, especially in columns: the population has grown, as shown in table 1 | a table of contents.
• Computing a collection of data stored in memory as a series of records, each defined by a unique key stored with it.

**Comment by rewolfe** [comment]
I think all we need to do is remove "A list", so it becomes

"Facts or or numbers arranged in a special order, usually in rows and
columns."

On Fri, Jul 24, 2015 at 12:40 AM, Stephan Zednik notifications@github.com
wrote:

> Current class definition:
> 
> gcis:Table a owl:Class ;
>     rdfs:label "Table" ;
>     rdfs:comment "A list of facts or numbers arranged in a special order, usually in rows and columns." ;
>     rdfs:subClassOf prov:Entity .
> 
> The second definition of 'table' from the New Oxford American Dictionary
> aligns pretty closely with the definition we have been using. I am fine
> with keeping what we have.
> 
> table |ˈtābəl|
> noun
> 1 a piece of furniture ...
> 2 a set of facts or figures systematically displayed, especially in
> columns: the population has grown, as shown in table 1 | a table of
> contents.
> • Computing a collection of data stored in memory as a series of records,
> each defined by a unique key stored with it.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/USGCRP/gcis-ontology/issues/82#issuecomment-124326908
> .

## 

Robert Wolfe, NASA GSFC @ USGCRP, o: 202-419-3470, m: 301-257-6966

**Comment by justgo129** [comment]
Fine with me.  Let's proceed with the pull request.

**Comment by justgo129** [comment]
Closed #82 on account of merged https://github.com/USGCRP/gcis-ontology/pull/94.
